.github/workflows/call-windows-unit-tests.yaml

```diff
@@ -0,0 +1,180 @@
+---
+name: Reusable workflow to run unit tests on Windows packages (only for x86 and x64)
+
+on:
+  workflow_call:
+    inputs:
+      version:
+        description: The version of Fluent Bit to create.
+        type: string
+        required: true
+      ref:
+        description: The commit, tag or branch of Fluent Bit to checkout for building that creates the version above.
+        type: string
+        required: true
+      environment:
+        description: The Github environment to run this workflow on.
+        type: string
+        required: false
+      unstable:
+        description: Optionally add metadata to build to indicate an unstable build, set to the contents you want to add.
+        type: string
+        required: false
+        default: ''
+    secrets:
+      token:
+        description: The Github token or similar to authenticate with.
+        required: true
+      bucket:
+        description: The name of the S3 (US-East) bucket to push packages into.
+        required: false
+      access_key_id:
+        description: The S3 access key id for the bucket.
+        required: false
+      secret_access_key:
+        description: The S3 secret access key for the bucket.
+        required: false
+
+jobs:
+  call-windows-unit-test-get-meta:
```

Comment:
What's this for? It does not seem to be used.

Reply:
It is but for no reason, there's just a dependency.

Reply:
Ahh, yes. Just like a skeleton to be able to run this workflow....